slm_porosity/problem.inp

```diff
@@ -73,7 +73,7 @@ IMEXtype = 9                # 1=(111)L, 2=(121)L, 3=(122)A(1.0), 4=(233)A(0.732)
                             # if IMEX is used, user_rhs must be split and be defined for IMEXswitch >=0 and <=0 without using ELSE, or the results of integration will be wrong
 
 t_begin = 0    	#  tbeg  0.2500000e+00
-t_end = 30	#  tend  1.50000e+00
+t_end = 1e-3	#  tend  1.50000e+00
```

Comment:
You can use your own input file. Please, do not pollute this tutorial-purpose one.